frontend/src/components/commons/TabNavigation.jsx

```diff
@@ -29,12 +29,28 @@ const styles = {
   }
 };
 
+/* this function is used to disable specific tabs using an array of tab ids 
+this function is located here in order to be able to test it */
+export const isTabIdDisabled = (array, tabId) => {
```

Comment:
A simpiler way is:
```
export const isTabIdDisabled = (array, tabId) => {
  // check if the tabId exists in the array
  return array.indexOf(tabId) > -1;
};
```

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/251/commits/b25cbcc9d808533cac93562d871ee5b3f32e2848